### ISSUE TITLE
Support css variables for edge marker colours

### ISSUE THREAD

**Issue by dancras** [wbkd/react-flow#3220]
The current implementation of the edge markers makes it very difficult to use a css variable for its colour. These are quite common with design systems. Example edge config:

```
{
    style: { stroke: 'var(--primary-400)' },
    markerEnd: { type: MarkerType.ArrowClosed, color: 'var(--primary-400)' },
}
```

For clarity I'm referring to the Edge type and the EdgeMarker type defined [here](https://reactflow.dev/docs/api/edges/edge-options/#edge)

The style property on Edge supports css vars, but the color property for EdgeMarker is being used with the svg attributes. If instead they used the style attribute, css variables should be supported. Here's a proposal of the change I think is necessary:

**packages/core/src/container/EdgeRenderer/MarkerSymbols.tsx**

```
const ArrowSymbol = ({ color = 'none', strokeWidth = 1 }: SymbolProps) => {
  return (
    <polyline
      style={{
        stroke: color,
      }}
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth={strokeWidth}
      fill="none"
      points="-5,-4 0,0 -5,4"
    />
  );
};

const ArrowClosedSymbol = ({ color = 'none', strokeWidth = 1 }: SymbolProps) => {
  return (
    <polyline
      style={{
        stroke: color,
        fill: color,
      }}
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth={strokeWidth}
      points="-5,-4 0,0 -5,4 -5,-4"
    />
  );
};
```

**Comment by moklick** [comment]
Thanks, that's good feedback! We will change this for v12.

**Comment by moklick** [comment]
available in 11.9.0
